help.md

```diff
@@ -0,0 +1,901 @@
+# Table of Contents
+<div class="toc">
+
+- About  
+- Layers  
+  - Creating New Layers
+  - Saving and Loading Layers
+  - Creating Layers from Other Layers
+- Layer Controls
+  - Layer Information
+  - Sorting
+  - Filtering
+    - Platform Filter
+  - Color Setup
+    - Tactic Row Background
+    - Scoring Gradient
+  - Hiding Disabled Techniques
+  - Showing or Hiding Sub-techniques
+  - Configuring the Layout
+    - Side Layout
+    - Flat Layout
+    - Mini Layout
+    - Showing IDs and Names
+    - Aggregate Scores
+      - Showing or Hiding Aggregate Scores
+      - Counting Unscored Techniques
+      - Choosing an Aggregate Function
+  - Legend
+- Technique Controls
+  - Disabling Techniques
+  - Assigning Manual Colors
+  - Scoring Techniques
+  - Adding Comments to Techniques
+  - Clearing Annotations on Techniques
+- Selecting Techniques
+  - Selecting with the Mouse
+  - Selection Behavior
+  - Search & Multiselect Interface
+- Customizing the Navigator
+  - Default Layers
+  - Disabling Features
+- Rendering Layers as SVG
+  - Measurement Units
+  - Configuring Image Size
+  - Configuring Text
+  - Customizing the Legend
+  - Display Settings
+- Exporting Layers to MS Excel
+- Notice
```

Comment:
Without hyperlinks, this table of contents isn't particularly helpful. It appears as though it does have links in the rendered page, but [not but when viewed on GitHub](https://github.com/mitre-attack/attack-navigator/blob/feature/%23179-help-page/help.md#table-of-contents).
- On GitHub TOCs are automatically generated:
   ![image](https://user-images.githubusercontent.com/28868888/124280171-5cf11900-db16-11eb-9174-292013381249.png)
- The TOC for the rendered markdown page could be inferred from the heading tags (see [this code example](https://github.com/center-for-threat-informed-defense/attack-workbench-frontend/blob/develop/app/src/app/views/help-page/help-page.component.ts#L30-L40)) and thereby auto-generated without needing to maintain a TOC in the markdown itself.